soh/src/code/z_play.c

```diff
@@ -745,6 +745,14 @@ void Play_Init(GameState* thisx) {
                     GET_PLAYER(play)->actor.world.pos.y + Player_GetHeight(GET_PLAYER(play)) + 5.0f,
                     GET_PLAYER(play)->actor.world.pos.z, 0, 0, 0, 1, true);
     }
+    if (gSaveContext.n64ddFlag && LINK_IS_ADULT &&
```

Comment:
Can we move this to a hook? If not I'd either extract this to a clearly named function or make a comment explaining what it's doing. Obvious to us now but 6 months from now "why are we spawning en_xc here?"

Reply:
I don't have the most experience with hooks or how they work here, but I'd be willing to move this to one.